Source/Scene/Scene.js

```diff
@@ -1876,6 +1876,9 @@ define([
         var environmentState = scene._environmentState;
         var renderPass = frameState.passes.render;
         environmentState.skyBoxCommand = (renderPass && defined(scene.skyBox)) ? scene.skyBox.update(frameState) : undefined;
+        if (defined(scene.skyAtmosphere) && defined(scene.globe)) {
```

Comment:
Since we are reusing `scene.skyAtmosphere` a few times here and below, it would be better to write this as
```javascript
var skyAtmosphere = scene.skyAtmosphere;
```
(same-ish for `scene.globe`).

See "Avoid redundant nested property access" here - https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#basic-code-construction